libmogwai-schedule/schedule-service.c

```diff
@@ -603,10 +620,21 @@ active_entries_changed_cb (MwsScheduler *scheduler,
   if (((added == NULL) != (removed == NULL)) ||
       (added != NULL && removed != NULL && added->len != removed->len))
     {
-      notify_scheduler_properties (self);
+      notify_scheduler_properties (self, TRUE, FALSE);
     }
 }
 
+static void
+allow_downloads_changed_cb (GObject    *obj,
+                            GParamSpec *pspec,
+                            gpointer    user_data)
+{
+  MwsScheduleService *self = MWS_SCHEDULE_SERVICE (user_data);
+
+  /* The com.endlessm.DownloadManager1.Scheduler property potentially changed */
+  notify_scheduler_properties (self, FALSE, TRUE);
```

Comment:
Yeah, it'd definitely be more clear when we see what we're telling it to notify :slightly_smiling_face: